src/Metadata/Resource/Factory/OperationResourceMetadataFactory.php

```diff
@@ -78,6 +78,10 @@ public function create(string $resourceClass): ResourceMetadata
 
             $resourceMetadata = $resourceMetadata->withItemOperations($this->createOperations($methods));
         } else {
+            if (!$itemOperations) {
```

Comment:
You must explicitly check for a get operation (only PUT could be defined for instance).

Reply:
how can this be done without links to the Router? I can move the `method` guess from ApiLoader to metadata first, but when using a `route_name`, we won't be able to check the method without retrieving the route from the RouteCollection right?

Reply:
> when using a route_name, we won't be able to check the method without retrieving the route from the RouteCollection right?

Applying the 80/20 rule, I'd say let's do nothing when we encounter such edge cases?